app/controllers/settings_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   def update
     @settings = current_user
     if @settings.update_attributes(settings_params)
+      SettingsMailer.update_email(@user).deliver_now
```

Comment:
There is no @user variable set!  This fails the tests and the click through. Be sure to run your tests before you submit the pull request!